File: cpp/Problem006_SumSquareDifference.cpp
#include <iostream>
#define MAX 100
using std::cout;

int main()
{
    long sumOfSquares = 0;
    long squareOfSum = 0;

    for (int i=1; i<=MAX; i++)
    {
        sumOfSquares += i*i;
        squareOfSum += i;
    }
    squareOfSum = squareOfSum * squareOfSum;

    cout << squareOfSum - sumOfSquares;
    return 0;
}

File: _try_out_langs_/rust/Problem006_SumSquareDifference.rs
fn main() {
    let mut sum_of_squares : i64 = 0;
    let mut square_of_sum : i64 = 0;
    for a in 1..=100 {
        square_of_sum += a;
        sum_of_squares += (a as i64).pow(2);
    }
    square_of_sum = square_of_sum.pow(2);

    println!("{}", square_of_sum - sum_of_squares);
}